flint.ui/src/components/Cards/CardInfoRun.vue

```diff
@@ -113,13 +111,17 @@ export default {
       axios
         .get(`${process.env.VUE_APP_REST_API_FLINT_EXAMPLE}/spec`)
         .then((response) => {
-          toast.success(`Specification route has been invoked.`, {
-            timeout: 2000
+          notification['success']({
```

Comment:
can you change all of the notification calls to `notification.success({...})` or `notification.error({...})`? 
The square bracket notation isn't necessary as the notification types aren't dynamic.